python/test/integration/test_quantile.py

```diff
@@ -4,25 +4,6 @@
 enable_features("floating-point", "contrib")
 
 
-def test_quantile_score_candidates():
```

Comment:
I moved this test to Rust. Now that the dtype is u64 and not usize, the function can't be directly called from our tests in Python.